api/grpc/mpi/v1/command.proto

```diff
@@ -0,0 +1,358 @@
+// These proto definitions follow https://protobuf.dev/programming-guides/style/
+// and recommendations outlined in https://static.sched.com/hosted_files/kccncna17/ad/2017%20CloudNativeCon%20-%20Mod%20gRPC%20Services.pdf
+syntax = "proto3";
+package f5.nginx.agent.api.grpc.mpi.v1;
+
+option go_package = "github.com/nginx/agent/v3/api/grpc/mpi/v1";
+
+import "common.proto";
+import "file.proto";
+import "google/protobuf/empty.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+
+// A service outlining the command and control options for a DataPlane
+service CommandService {
+    // Connects NGINX Agents to the Management Plane agnostic of instance data
+    rpc Connect(ConnectionRequest) returns (ConnectionResponse) {}
+    // Reports on instances and their configurations
+    rpc Status(DataPlaneStatus) returns (google.protobuf.Empty) {}
+    // Reports on instance health
+    rpc Health(DataPlaneHealth) returns (google.protobuf.Empty) {}
+    // A decoupled communication mechanism between the data plane and management plane.
+    rpc Subscribe(stream DataPlaneMessage) returns (stream ManagementPlaneMessage) {}
+}
+
+// The connection request is an intial handshake to establish a connection, sending NGINX Agent instance information
+message ConnectionRequest {
+    // Meta-information associated with a message
+    f5.nginx.agent.api.grpc.mpi.v1.common.MessageRequest message_metadata = 1;
```

Comment:
nit, `message_metadata`, we have "meta" in other places. Since metadata and meta are kind of the same thing, please pick one to be consistent.

Reply:
made consistent. Resolving